### PR TITLE
Add EIP list

### LINKED ISSUE #6: Document specific EIPs supported
To avoid situations like this: https://discuss.status.im/t/sending-limit-per-contact/1179/6?u=oskarth

And make it clear which once we support and not.

Also see LES etc.

This is more of an INFORMATIONAL one for each client (status-go, Status App), but really useful to have at a glance.

### PR DESCRIPTION
**Pull request by 3esmit**
Fixes #6

### CODE REVIEW

**Comment by 3esmit** [comment]
There might be missing some EIPs. Feel free to ask for any changes regarding format.

**Comment by oskarth** [comment]
This is awesome, thank you!

File: status-EIPs.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# Status EIPs standards
```
Comment (oskarth) [inline status-EIPs.md:1]: Could we add a link to this in the README and from the main client spec as well?

**Review by oskarth — COMMENTED** [review]
looks like a good start to me

File: status-EIPs.md (hunk 1 above)
Comment (3esmit) [reply status-EIPs.md:1]: Added to README. In main client spec I am not sure where this link fits.

**Comment by 3esmit** [comment]
I am not sure we want to call it EIP standards, but something more like "web3 standards", because it involves other then EIPs, such also BIPs and maybe in future other such as https://github.com/WalletConnect/WCIPs 

This document is intended to facilitate the interoperability of web3 in general, so describing all standards that Status follow. In case of EIPs, there is also some that were authored by Status, such as 1077 (gas abstraction), 1581 (non wallet keys) and 2400 (transaction receipt uri)

**Review by kdeme — COMMENTED** [review]
Noticed when looking at the markdown formatted text that `Support:`, `Reference:`, etc. are not separated by newlines. I think you need to add a bunch of double spaces to do so. Or format it some other readable way.

**Review by decanus — APPROVED** [review]
this is great thanks!

File: status-EIPs.md (hunk 1 above)
Comment (oskarth) [reply status-EIPs.md:1]: Added commit with reference

File: status-EIPs.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  - [Security Considerations](#security-considerations)
+  - [Design Rationale](#design-rationale)
+  - [Footnotes](#footnotes)
+  - [Acknowledgements](#acknowledgements)
+
+## Introduction
+
+Status should follow all standards as possible. Whenever a feature is needed, it should be first checked if there is a standard for that, if not, Status should propose a standard.
+
+### Support table
```
Comment (oskarth) [inline status-EIPs.md:26]: A bit hard to horizontally scroll, I wonder if it makes sense to flip these axes around?

**Review by oskarth — APPROVED** [review]
Fixed the client spec mention, I'd be fine to merge this

File: status-EIPs.md (hunk 2 above)
Comment (3esmit) [reply status-EIPs.md:26]: Good idea. Just transposed the table. I used this https://www.tablesgenerator.com/markdown_tables

**Review by kdeme — APPROVED** [review]

**Pull request merged**